eclass/java-utils-2.eclass

```diff
@@ -1368,34 +1353,6 @@ java-pkg_register-environment-variable() {
 	java-pkg_do_write_
 }
 
-# @FUNCTION: java-pkg_get-bootclasspath
-# @USAGE: <version>
-# @DESCRIPTION:
-# Returns classpath of a given bootclasspath-providing package version.
-#
-# @param $1 - the version of bootclasspath (e.g. 1.5), 'auto' for bootclasspath
-#             of the current JDK
-java-pkg_get-bootclasspath() {
```

Comment:
this one is still called by another eclass:
```
eclass/java-ant-2.eclass:       local bcp="$(java-pkg_get-bootclasspath "${version}")"
```
i did not look at that eclass but maybe dropping just the `1.5` case here would be the right solution.

Reply:
Sorry, I should have looked at it before deleting.
Done.